src/DocumentTitle.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import useTitle from 'react-use/lib/useTitle'
+import { usePriceCakeBusd } from './state/hooks'
+
+const DocumentTitle = ({ title, showPrice = true }) => {
+  const documentTitle = []
+
+  // Append price if enabled
```

Comment:
Try and use comments only if the code needs supplemental explanation. I think this case it is pretty obvious what is happening.